src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/JS/WasmTestCommand.cs

```diff
@@ -113,6 +113,8 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
             }
 
             engineArgs.AddRange(PassThroughArguments);
+            // Node respects LANG only, ignores LANGUAGE
+            Environment.SetEnvironmentVariable("LANG", Arguments.Locale);
```

Comment:
It's also possible to set the env just for the process you're launching with `processManager` below. Maybe that's better? Or do you need this for other processes you don't control?

Reply:
Only for the process below, should get changed, thanks